include/MainWindow.h

```diff
@@ -81,9 +81,20 @@ class MainWindow : public QMainWindow
 	///
 	bool mayChangeProject(bool stopPlayback);
 
-	void autoSaveTimerStart()
+	// Auto save timer intervals
+	const static int m_autoSaveLongSeconds = 120;
+	const static int m_autoSaveShortSeconds = 10;
+	const static int m_autoSaveLongTime = m_autoSaveLongSeconds * 1000;
+	const static int m_autoSaveShortTime = m_autoSaveShortSeconds * 1000;
```

Comment:
Redundant with my other remarks, but I don't think this feature needs 4 new constants.  Let's see if we can boil it down to 1 or 2. :+1: